Source/WebCore/animation/AnimationTimelinesController.cpp

```diff
@@ -506,18 +506,24 @@ void AnimationTimelinesController::unregisterNamedTimeline(const AtomString& nam
 
 void AnimationTimelinesController::setTimelineForName(const AtomString& name, const Element& element, WebAnimation& animation)
 {
+    ALWAYS_LOG_WITH_STREAM(stream << " AnimationTimelinesController::setTimelineForName: animation: " << &animation);
+
     auto it = m_nameToTimelineMap.find(name);
     if (it == m_nameToTimelineMap.end()) {
+        animation.setIsPendingTimelineAttachment(true);
```

Comment:
Since we already have a list of animations pending attachment on `AnimationTimelinesController`, could we not use this to query whether the animation is pending attachment instead of adding another bit on `WebAnimation`?